Common/UnitDefinitions/CloudCover.json

```diff
@@ -0,0 +1,43 @@
+{
+  "Name": "CloudCover",
+  "BaseUnit": "Fraction",
+  "XmlDoc": "Cloud cover (also known as cloudiness, cloudage, or cloud amount) refers to the fraction of the sky obscured by clouds when observed from a particular location.",
+  "Units": [
+    {
+      "SingularName": "Okta",
+      "PluralName": "Oktas",
+      "FromUnitToBaseFunc": "x * 0.125",
+      "FromBaseToUnitFunc": "System.Math.Round(x / 0.125, MidpointRounding.AwayFromZero)",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "oktas" ]
```

Comment:
Apparently there are unicode symbols for the okta values, maybe we should offer a method to get the unicode string representation for the cloud cover?

https://en.wikipedia.org/wiki/Okta

This can be added to `UnitsNet/CustomCode/Quantities/CloudCover.extra.cs` as an additional method
```c#
public string ToOktaSymbol() 
{
    // 50% cloud cover => ◑
    // etc.
}
```

Reply:
I was working on this, actually doing it as a char property with a getter but then I noticed there aren't Unicode characters for all oktas; 3, 5 and 7 are missing. Not sure we should go there. The full symbols can be found at https://en.wikipedia.org/wiki/Okta#/media/File:Cloud_Cover_in_Oktas.svg but as you see we don't have Unicode values for 3, 5 and 7.

Reply:
We could just round to the nearest available symbol, but I understand if this would perhaps not satisfy real use.

Reply:
No, it won't. if you have 3 oktas, you don't want to see the symbol of either 2 or 4. Best we could do would be output the SVG for displaying the symbol, but I think this would go way out of the scope of UnitsNet.